android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -227,6 +228,22 @@ public void onServiceDisconnected(ComponentName name) {
         customTabsSupported = bindService(customTabIntent, customTabsServiceConnection, Context.BIND_AUTO_CREATE);
     }
 
+    private void setUserProfileMenuItem(){
+        MenuItem userProfileMenuItem = navigationView.getMenu().findItem(R.id.nav_user_profile);
+        if (AuthUtil.isUserLoggedIn()) {
+            String email = SharedPreferencesUtil.getString(ConstantStrings.USER_EMAIL, null);
+            String firstName = SharedPreferencesUtil.getString(ConstantStrings.USER_FIRST_NAME, null);
+            String lastName = SharedPreferencesUtil.getString(ConstantStrings.USER_LAST_NAME, null);
+
+            if (firstName != null && lastName != null && !TextUtils.isEmpty(firstName) && !TextUtils.isEmpty(lastName))
```

Comment:
TextUtils.isEmpty also checks for null, no need to do it explicitly. Also, only check for isEmpty(firstName) and is it is not empty, show firstName + " " + lastName

Showing only firstName if lastName is null is redundant as `X + " " + null` is `X`

Use isEmpty for email check as well

Reply:
done with the requested changes